src/software/full_system_main.cpp

```diff
@@ -52,14 +53,23 @@ commandLineArgs parseCommandLineArgs(int argc, char **argv)
     std::string backend_help_str =
         "The backend that you would like to use, one of: " + all_backend_names;
 
+    std::string interface_help_str =
+        "The interface to send and receive packets over (can be found through ifconfig)";
+
     boost::program_options::options_description desc{"Options"};
     desc.add_options()("help,h", boost::program_options::bool_switch(&args.help),
-                       "Help screen")(
+                       "Help screen");
+    desc.add_options()(
         "backend",
         boost::program_options::value<std::string>(&args.backend_name)->required(),
-        backend_help_str.c_str())("headless",
-                                  boost::program_options::bool_switch(&args.headless),
-                                  "Run without the FullSystemGUI");
+        backend_help_str.c_str());
+    desc.add_options()(
+        "interface",
+        boost::program_options::value<std::string>(&args.network_interface_name)
+            ->required(),
+        interface_help_str.c_str());
```

Comment:
is this required even when backend is radio?

Reply:
I think it should be an optional override, where the default is whatever is in dynamic params

Reply:
I'm not sure that there's a common enough interface name for our usages to justify having a default dynamic param value for this?

Reply:
it has a default value already, I think Jons point was more about not specifying interface for radio, fixed to make it optional